src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsBtfBlocking.java

```diff
@@ -52,9 +48,7 @@ public void adsAtfDisableBtfOasis() {
     Assertion.assertTrue(adsBaseObject.checkSlotOnPageLoaded(AdsContent.TOP_LB), "Slot" + AdsContent.TOP_LB);
     Assertion.assertTrue(adsBaseObject.checkSlotOnPageLoaded(AdsContent.INVISIBLE_SKIN), "Slot" + AdsContent.INVISIBLE_SKIN);
 
-    // functionality tested below will be fixed in the scope of: https://wikia-inc.atlassian.net/browse/ADEN-6028
     Assertion.assertFalse(adsBaseObject.checkSlotOnPageLoaded(AdsContent.BOTTOM_LB), "Slot: " + AdsContent.BOTTOM_LB);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Slot: " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)